src/framework/COScript.m

```diff
@@ -356,19 +356,20 @@ - (id)require:(NSString *)module {
     
     id result = nil;
     
-    if ([module characterAtIndex:0] == '.') { // relative path
+    if ([module characterAtIndex:0] == '.' || [module characterAtIndex:0] == '/') { // relative or absolute path
         NSFileManager *fileManager = [NSFileManager defaultManager];
+        BOOL isRelative = [module characterAtIndex:0] == '.';
         NSString* modulePath = module;
         
-        // path/to/module/index.js
-        NSURL* moduleDirectoryURL = [NSURL URLWithString:[module stringByAppendingPathComponent:@"index.js"] relativeToURL:currentURL];
+        // /path/to/module/index.js
```

Comment:
The comment change doesn't accurately capture the code, the code now deals with either. Same goes for the comment change below.